routing/pathfind_test.go

```diff
@@ -2756,11 +2775,14 @@ func testProbabilityRouting(t *testing.T, useCache bool,
 
 		switch {
 		case fromNode == alias["a1"] && toNode == alias["a2"]:
-			return p10
+			return p10 * capacityFactor(amt, capacity)
```

Comment:
I would be careful with using `capacityFactor` in expectations, because it basically excludes its implementation from the coverage. I know you've covered it separately, but still.

Reply:
The probability expectations are now hardcoded. This test here may not be that useful as discussed, could remove it.

Reply:
Yes, I'd remove it here.